lib/serverless-tools/deployer/r_builder.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ServerlessTools
+  module Deployer
+    class RBuilder
```

Comment:
Is this an R builder or Docker builder?

Reply:
Yeah, reading through the final code, there definitely wasn't a need to make this only R-specific. Initially I thought there may be some logic specific to R setup, but luckily that's all encapsulated in the Dockerfile. I've updated it now 👍